handler/handler_test.go

```diff
@@ -148,20 +126,14 @@ func TestHandleWithTwoEventsBothEventCreated(t *testing.T) {
 	})
```

Comment:
On line 121, 
The Convey message should be 
`Then the error is nil`

The remainder `only create bulk is called but not update bulk request` is incorrect as only one call "upsert" is always made

Also on line 123, the test to check `count` is `1` does not inform what call is made. It only informs whether a call is made or not

Reply:
Please note that count will be useful if you are testing that a call has been made to ES, irrespective of the type of call